views/support/faqsearch.tt

```diff
@@ -0,0 +1,53 @@
+[%- CALL dw.active_resource_group("foundation") -%]
+[%- dw.need_res({ group => "foundation"}
+        "stc/faq.css"
+        ) -%]
+
+[%- sections.title = dw.ml('.title') -%]
+
+<p>[% '.info' | ml %]</p>
+<form method='GET'>
+    <div class="columns">
+        <div class="row">
+            <div class="columns large-3 text-right">[% '.label.term' | ml %]:</div>
+            <div class="columns large-6">
+                [%- form.textbox( size = 30, value = q, name = 'q') -%]
+            </div>
+            <div class="columns large-3">
+                [%- form.submit( 'value' = dw.ml('.button.search')) -%]
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="columns large-3 text-right">
+                [% '.label.lang' | ml %]
+            </div>
+            <div class="columns large-3">
+                [%- form.select( name = 'lang', selected = sel, items = langs) -%]
+            </div>
+            <div class="columns large-6">
+                [% '.example' | ml %]
+            </div>
+        </div>
+    </div>
+</form>
+[%- IF q && q.length < 1 -%]
+<p><i>[% '.error.noterm' | ml %]</i></p>
+[%- END -%]
+
+[%- IF q && q.length < 2 -%]
+<p><i>[% '.error.tooshort' | ml %]</i></p>
+[%- END -%]
+
+[%- IF results -%]
+    [%- IF results.size < 1 -%]
+        <p><i>[% '.error.noresults' | ml %]</i></p>
+    [%- ELSE -%]
+
+    <ul class='spaced'>
+        [%- FOR r IN results -%]
+            <li><a href='/support/faqbrowse?faqid=[% r.id %]&q=[% r.ueq _ r.ul %]'>[% r.dq %]</a></li>
```

Comment:
What's that underscore operator in `r.ueq _ r.ul` ?

Reply:
That's Template Toolkit's concatenate operator, for whatever reason.

Reply:
💀